src/Tribe/Editor/Blocks/Rsvp.php

```diff
@@ -235,7 +235,12 @@ public function assets() {
 			$plugin,
 			'tribe-tickets-rsvp-manager',
 			'v2/rsvp-manager.js',
-			[ 'jquery', 'tribe-common', 'tribe-tickets-rsvp-block' ],
+			[
+				'jquery',
+				'tribe-common',
+				'tribe-tickets-rsvp-block',
+				'tribe-tickets-rsvp-tooltip'
```

Comment:
* :no_entry_sign: Each array item in a multi-line array declaration must end in a comma
* :no_entry_sign: Comma required after last value in array declaration

<sub>hash: d4a2dd56d1ff71e55f06623470eb6be8</sub>